LinkResolverService.cs

```diff
@@ -127,6 +134,11 @@ public class LinkResolverService
 
             string href = UrlResolver.Service.GetUrl(pageData);
 
+            if (!string.IsNullOrWhiteSpace(languageBranchId))
```

Comment:
Similar to a previous comment. Can we make this something like:
`string href = string.IsNullOrWhiteSpace(languageBranchId) ? UrlResolver.Service.GetUrl(pageData) : UrlResolver.Service.GetUrl(pageData.ContentLink, languageBranchId);`?